Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.8-slim as build
+
+WORKDIR /code
+
+RUN pip install --no-cache-dir 'poetry==1.1.6'
+
+COPY poetry.lock pyproject.toml /code/
+
+RUN poetry config virtualenvs.create false \
+  && poetry install --no-dev --no-interaction --no-ansi
+
+FROM python:3.8-alpine
```

Comment:
In this case wouldn't it be better to use the same image on the build and final stages?

Reply:
Yeah, I was in doubt if this would generate some problem, but apparently everything worked.
I did this to make the image smaller.

Using the `alpine` in the build stage didn't work, we get compilation errors.

Since everything worked, I thought this was a safe choice.

Reply:
But I can spend some more time trying to figure out if we can fix the compilation erros in alpine, which I didn't do yet.

Reply:
Done, alpine just needed some additional dependencies to be able to compile things